README.md

```diff
@@ -89,3 +89,14 @@ Default value: ```0```
 Default value: ```False```
 
 Note: Although multithreading provides a faster publishing rate on certain platforms, on less performant Hosts this might not be beneficial and the single thread implementation can be more relevant.
+
```

Comment:
Can here be added **ROS Parameters** as main title and a subsection to be the thread parameters? There are other parameters that might need to be added here..